recipes/winmd/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES C) # if the project is pure C
+project(test_package LANGUAGES CXX) # if the project uses c++
```

More cleanup form tempalte copy pasting

```suggestion
project(test_package LANGUAGES CXX)
```